### DISCUSSION TITLE
Introduction and a few questions

### DISCUSSION THREAD

**Discussion by ImperaLuna** [MDAnalysis/mdanalysis#4453] [GSoC Q&A] [answered]
Hey, my name is Rares I live in Romania and I’m a first-year student in computer science. I’ve been learning python for about 7-8 months. I’m interested in applying for GSoC 2024. 
I also attended 4 years of medical university and have some basic understanding of organic chemistry (high school level) & biochemistry (university), but I never had hands on experience with anything related to molecular dynamics.

I have a few questions I hope you can answer:
1.	Considering my experience, do you think I’d be a good candidate for your organization?
2.	Not sure if this is the right place to ask, I’ve read on your blog from https://www.mdanalysis.org/ about a workshop, I was wondering if I could apply to it either as a passive participant or if it would be recorded. It seems like a great place to understand more about MDAnalysis and its usage, but I wouldn’t want to take up one of the limited spots unnecessarily.
3.	I was wondering, in case either the organization is not selected for GSoC, or I’m not selected for the program, would it still be possible to work on one of the project ideas? Or are they designed specifically for the program. I was thinking either project 4 or 5 would be the best fit for my skill.

**Comment by RMeli** [comment, marked as answer]
Hello @ImperaLuna, welcome to MDAnalysis! It's great to hear you are interested in the project.

1. We haven't finalised the requirements for this year (we don't know if we will be selected), but in recent years we started [requiring](https://github.com/MDAnalysis/mdanalysis/wiki/Google-Summer-Of-Code#prerequisites) "basic knowledge and hands-on experience of molecular dynamics simulations and the associated analyses, or equivalent experience in simulations and modelling of molecular system". Since MDAnalysis is a package for the analysis of molecular simulation, familiarity with the field is essential to work on most of the library. That said, mentors for this year might have more precise requirements in mind (I'll let them chime in), and there are a lot of molecular dynamics tutorial around, so it is entirely possible to pick up this knowledge informally. On  [Learning MDanalysis](https://www.mdanalysis.org/pages/learning_MDAnalysis/) you have some introductory material which covers also a bit of MD. For MDAnalysis specifically, going through the [User Guide](https://userguide.mdanalysis.org/stable/index.html) will teach you a whole lot!
2. I believe the workshop will be recorded and released on the [MDAnalysis YouTube Channel](https://www.youtube.com/channel/UC3TCuK-z_bJNdwWCvsH9D3Q). There, you will find videos of past workshops and talks too.
3. If the organisation is not selected for GSoC, then it will be possible for you to work on the project ideas. They are cool stuff we would like to have in the library, regardless of how they are done. However, there will be no formal mentorship, so there will likely be little guidance (you will have to interact with the community as usual, via Discussions, Issues and PRs). If we are selected for GSoC but you are not selected as a participant, it will be possible to work on projects that have not be assigned to the selected participants. But again, there will likely be little guidance, since the mentors will be mostly focussing on the selected GSoC projects.

**Reply by ImperaLuna** [reply]
Thanks for answering my questions, and for providing resources. I’ll do my best to get more familiar with the field and with the codebase in the following weeks.
